lib/ansible/modules/files/stat.py

```diff
@@ -44,7 +50,7 @@
     description:
       - Algorithm to determine checksum of file. Will throw an error if the
         host is unable to use specified algorithm.
```

Comment:
Need a note that the remote host has to support that hashing method (md5 can be unavailable if the remote machine is running in FIPS mode)

Reply:
Have added this info to the description.